### PR TITLE
Fix po notes section (header layout + theming) & folders/files naming conventions

### PR DESCRIPTION
**Pull request by samim-hossain-mondal**
- fix po-notes-header layout (search, quick filters div)
  * shift search icon inside the search bar
  * keep search bar and quick filter comp in the same div
  * fix some other minor spacings
- add theming (colors) for whole po-notes section
- removed extra spaces in the codes
- folders/files naming conventions & some minor bugs
  * correct structure and naming of folders/files but faced issue* while pushing 

**Issue:** _Initially git didn't recognise the naming convention changes of folders and files, so I have to set git config. ignoreCase to false then push it, but it creates new files for all resulting to duplicates one with earlier inconsistent naming conventions and other new one. You can check the commits for the same. So, accordingly merge it if approves else don't._ 

_ps:_ working in other prev review comments.

Snapshot for the current one -
Screenshot 2023-02-10 at 5 56 31 PM

### CODE REVIEW

**Review by Harshit-Soni-Mck — COMMENTED** [review]
quick filters placeholder in not centre aligned as search input is, looks kind of odd

**Comment by samim-hossain-mondal** [comment]
> quick filters placeholder in not centre aligned as search input is, looks kind of odd

okay got it.

**Comment by samim-hossain-mondal** [comment]
> quick filters placeholder in not centre aligned as search input is, looks kind of odd

rectified and pushed 👍  (in fix/poNotes-headerLayout branch)

**Comment by samim-hossain-mondal** [comment]
> > quick filters placeholder in not centre aligned as search input is, looks kind of odd
> 
> rectified and pushed 👍 (in fix/poNotes-headerLayout branch)
Screenshot 2023-02-11 at 2 50 45 AM

File: src/components/cards/CustomCard.js [hunk 1]
```diff
@@ -23,33 +32,16 @@ function toggle(value) {
...
+export default function CustomCard({ chckBox, data, type }) {
   const [checked, setChecked] = useState(false);
-  // const classes = useStyles();
 
   const renderdueDate = () => {
-    if (type==='action_item') {
-         return <Typography color="primary" fontWeight={500} mt={2}> Needed By {dateGetter(data.dueDate)}</Typography>
-    } 
-    return <Typography color="primary" fontWeight={500} mt={2} sx={{visibility: 'hidden' }}> Needed By {dateGetter(data.dueDate)}</Typography>
+    if (type === 'action_item') {
```
Comment (sugumar-mani) [inline src/components/cards/CustomCard.js:39]: Why not declare enum / constant for these static strings? if you have already made these changes in another PR then ignore this comment

File: src/components/poNotesComponents/AddPoNotes.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    handleSubmit('COMPLETED');
+  };
+  const handleNoteType = (event) => {
+    setNoteType(event.target.value);
+  };
+  const handleStatement = (event) => {
+    setStatement(event.target.value);
+  };
+  return (
+    <Box sx={{ flexGrow: 0.2, display: { xs: 'none', md: 'flex' } }}>
```
Comment (sugumar-mani) [inline src/components/poNotesComponents/AddPoNotes.js:63]: I am not sure about how responsive is our current UI for various device dimensions. Please consider making it responsive as you go through Code academy classes.

**Review by sugumar-mani — APPROVED** [review]
Please consider my comments. If you feel you can fix these in separate PR please go ahead.

**Pull request merged**